docs/resources.md

```diff
@@ -32,9 +32,68 @@ To view the list of pods it has annotated:
 kubetl get pods -l kritis.grafeas.io/invalidImageSecPolicy=invalidImageSecPolicy
 ```
 
+## GenericAttestationPolicy CRD
+
+GenericAttestationPolicy (GAP) is a Custom Resource Definition which enforces policies based on pre-existing attestations.
+The policy expects all attestation authorities to be satisfied before allowing the container image to be admitted.
+As opposed to [ISPs](#imagesecuritypolicy-crd) the GAP does not create new attestations.
+The general use case for GAPs are to have a policy that enforces attestations that have come from your CI pipeline, or other places in your release pipeline.
+
+ - Kubernetes Custom resource definition
+ - requires pre-existing attestations
+ - Expects all attestation authorities to be satisfied
+ - does not create attestations
```

Comment:
This formatting looks odd, are the `-`s added by accident?

Reply:
Oops, that wasn't meant to make its way in. It was just me planning the above lines. Have removed